src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -75,6 +98,15 @@ std::shared_ptr<AbstractIOHandler> createIOHandler<json::TracingJSON>(
         return constructIOHandler<ParallelHDF5IOHandler, openPMD_HAVE_HDF5>(
             "HDF5", path, access, comm, std::move(options));
     case Format::ADIOS1:
+        if (auxiliary::getEnvNum(suppressAdios1DeprecationWarning, 0) == 0)
+        {
+            int rank;
+            MPI_Comm_rank(comm, &rank);
```

Comment:
Maybe guard with an [MPI_Initialized](https://www.mpich.org/static/docs/v3.1/www3/MPI_Initialized.html) check?
If false, then just print. if true, then only print from rank 0

Reply:
I was not aware that such a check is possible. We can do that, but I think that we don't need to (see comment above).

Reply:
Added a comment above: I think we use the wrong (serial) runner for the MPI Core tests and should split them.

Reply:
I've moved the parallel part to the ParallelIOTests now